.commitlintrc.yml

```diff
@@ -0,0 +1,82 @@
+# Commitlint configuration.
+# See: https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md
+---
+rules:
+  body-leading-blank:
+    - 2
+    - always
+  body-max-line-length:
+    - 2
+    - always
+    - 75
+  footer-leading-blank:
+    - 2
+    - always
+  header-max-length:
+    - 2
+    - always
+    - 75
+  scope-enum:
+    - 2
+    - always
+    - - advisor
+      - analyzer
+      - cli
+      - clients
+      - deps
+      - detekt
+      - docker
+      - downloader
+      - evaluator
+      - examples
+      - github
+      - helper
+      - integrations
+      - model
+      - notifier
+      - reporter
+      - scanner
+      - utils
+  scope-case:
+    - 2
+    - always
+    - lower-case
+  subject-case:
+    - 2
+    - always
+    - - pascal-case
+      - sentence-case
+      - start-case
+      - upper-case
+  subject-empty:
+    - 2
+    - never
+  subject-full-stop:
+    - 2
+    - never
+    - .
+  type-case:
+    - 2
+    - always
+    - kebab-case
+  type-empty:
+    - 2
+    - never
+  type-enum:
+    - 2
+    - always
+    - - build
+      - chore
+      - ci
+      - docs
+      - feat
+      - fix
+      - perf
+      - refactor
+      - revert
+      - style
+      - test
+      - ui
```

Comment:
Why do you include `ui`? It's not in the default Angular types: https://github.com/conventional-changelog/commitlint/tree/master/@commitlint/config-conventional#type-enum

Reply:
And maybe we should use `deps:` instead of `chore(deps):` for dependency updates, could make it a bit easier to have a separate section in the changelog for dependency udpates. E.g. jreleaser by default only groups by type, not by scope, but that's configurable of course:
https://github.com/jreleaser/jreleaser/blob/ddc4d3545ea65a6947890aa0415770677a863c37/core/jreleaser-model-impl/src/main/resources/META-INF/jreleaser/changelog/preset-conventional-commits.yml

Reply:
> Why do you include `ui`?

I was thinking of updates to assets like logos. Where else would that fit?

Reply:
> And maybe we should use `deps:` instead of `chore(deps):` for dependency updates

Fine with me, esp. as dependency updates could be more important than "just a chore", but that would require some Renovate-config changes, right?

Reply:
> I was thinking of updates to assets like logos. Where else would that fit?

ORT doesn't have a UI, so maybe `chore(assets)`?

> Fine with me, esp. as dependency updates could be more important than "just a chore", but that would require some Renovate-config changes, right?

Yes, see: https://docs.renovatebot.com/semantic-commits

Reply:
> ORT doesn't have a UI, so maybe `chore(assets)`?

That should work. I've removed `ui` again.

> Yes, see: https://docs.renovatebot.com/semantic-commits

:+1: